diff --git a/modules/codec/avcodec/chroma.c b/modules/codec/avcodec/chroma.c
--- a/modules/codec/avcodec/chroma.c
+++ b/modules/codec/avcodec/chroma.c
@@ -183,10 +183,5 @@ static const struct
 #if LIBAVUTIL_VERSION_CHECK(52, 10, 0, 25, 100)
     {VLC_CODEC_XYZ12, AV_PIX_FMT_XYZ12, 0xfff0, 0xfff0, 0xfff0},
 #endif
-
-#ifdef HAVE_OHOS
-    {VLC_CODEC_OHOS_OPAQUE, AV_PIX_FMT_OHOSCODEC, 0, 0, 0},
-#endif
-
     { 0, 0, 0, 0, 0 }
 };
